.github/workflows/pr-smoke-test.yml

```diff
@@ -96,8 +96,10 @@ jobs:
             await script({github, context})
 
       - name: Remove label from pull request.
-        if: always() && ${{ contains( github.event.pull_request.labels.*.name, format('run{0} smoke tests', ':')) }}
+        if: |
+          always() 
+           && contains( github.event.pull_request.labels.*.name, format('run{0} smoke tests', ':')) 
         uses: actions-ecosystem/action-remove-labels@v1
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          labels: 'run: smoke tests'
+          labels: 'run: smoke tests'   
```

Comment:
@MrJnrman Nice work on this. I see from the activity on the PR that this tested great. The only thing that needs to be addressed is the trailing space or tab at the end of this line.

Reply:
Thanks Ron. The white space has been removed.